### PR TITLE
Add notebook with examples for the RiskEstimators

### PR DESCRIPTION
**Pull request by PanPip**
This notebook provides explanations of the algorithms from the RiskEstimators class, which are related do different ways of calculation of the covariance matrices and their adjustments. It also has examples of how the functions from the mlfinlab package can be used and the analysis of the results.

### CODE REVIEW

File: RiskEstimators/RiskEstimators.ipynb [hunk 1]
```diff
@@ -0,0 +1,2756 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "_\"This shrinkage is done by shifting every eigenvalue according to a given offset, which is equivalent to finding the l2-penalized Maximum Likelihood Estimator of the covariance matrix\"._\n",
+    "\n",
+    "_\"Shrinkage boils down to a simple a convex transformation\":_\n",
+    "\n",
+    "$$\\sum_{shrunk} = (1 - \\alpha)\\hat\\sum + \\alpha\\frac{Tr \\hat\\sum}{p}Id$$\n",
```
Comment (aditya1702) [inline RiskEstimators/RiskEstimators.ipynb:985]: the \\hat does not seem to load properly. Please look into that.

File: RiskEstimators/RiskEstimators.ipynb [hunk 2]
```diff
@@ -0,0 +1,2756 @@
...
+    "\n",
+    "This measure can be used to decrease the negative volatility and is being more precise for this goal than the covariance matrix that measurs both positive and negative variance. \n",
+    "\n",
+    "According to the __Minimum Downside Volatility Indices__ paper:\n",
+    "\n",
+    "_\"Each element in the Semi-Covariance matrix is calculated as:\"_\n",
+    "\n",
+    "$$SemiCov_{ij} = \\frac{1}{T}\\sum_{t=1}^{T}[Min(R_{i,t}-B,0)*Min(R_{j,t}-B,0)]$$\n",
+    "\n",
+    "_where $T$ is the number of observations, $R_{i,t}$ is the return of an asset $i$ at time $t$, and $B$ is the threshold return._\n",
```
Comment (aditya1702) [inline RiskEstimators/RiskEstimators.ipynb:1496]: $R_{i,t}$ is not rendered properly in the notebook

File: RiskEstimators/RiskEstimators.ipynb [hunk 3]
```diff
@@ -0,0 +1,2756 @@
...
+    "\n",
+    "_If the $B$ is set to zero, the volatility of negative returns is measured._\n",
+    "\n",
+    "A deeper analysis of use cases of Semi-Covariacne matrix is available in the paper by _Solactive AG - German Index Engineering_ __Minimum Downside Volatility Indices__ [available here](https://www.solactive.com/wp-content/uploads/2018/04/Solactive_Minimum-Downside-Volatility-Indices.pdf)"
```
Comment (aditya1702) [inline RiskEstimators/RiskEstimators.ipynb:1500]: Spelling of Covariance is wrong

File: RiskEstimators/RiskEstimators.ipynb [hunk 4]
```diff
@@ -0,0 +1,2756 @@
...
+   "source": [
+    "print('The Simple Covariance is:')\n",
+    "cov_matrix"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the results it's seen that the variance of the EWG has increased in the last observation, whereas the the variacne of the EEM has decreased. \n",
```
Comment (aditya1702) [inline RiskEstimators/RiskEstimators.ipynb:2109]: Spelling of variance is wrong

File: RiskEstimators/RiskEstimators.ipynb [hunk 5]
```diff
@@ -0,0 +1,2756 @@
...
+    "\n",
+    "Many other conclusions can be drawn also from the covariances, such as in the last observations the covariance between the EEM and the EWJ has decreased."
```
Comment (aditya1702) [inline RiskEstimators/RiskEstimators.ipynb:2111]: Can you write this in a more explanatory manner - I did not understand the sentence - "such as in the last observations the covariance between the EEM and the EWJ has decreased."

File: RiskEstimators/RiskEstimators.ipynb (hunk 1 above)
Comment (PanPip) [reply RiskEstimators/RiskEstimators.ipynb]: Removed hats, as they won't render above the Sum. Replaced with "unshrinked" sign.

File: RiskEstimators/RiskEstimators.ipynb (hunk 2 above)
Comment (PanPip) [reply RiskEstimators/RiskEstimators.ipynb]: Done

File: RiskEstimators/RiskEstimators.ipynb (hunk 3 above)
Comment (PanPip) [reply RiskEstimators/RiskEstimators.ipynb]: Fixed

File: RiskEstimators/RiskEstimators.ipynb (hunk 4 above)
Comment (PanPip) [reply RiskEstimators/RiskEstimators.ipynb]: Fixed

File: RiskEstimators/RiskEstimators.ipynb (hunk 5 above)
Comment (PanPip) [reply RiskEstimators/RiskEstimators.ipynb]: Rewrote with better explanation. Thank you!

**Pull request merged**